platform/ui/src/components/Modal/Modal.jsx

```diff
@@ -41,20 +54,32 @@ const Modal = ({
 
   return (
     <ReactModal
-      className={classNames(className)}
+      className={classNames(
```

Comment:
Why does the user need a way to pass className?

Reply:
It was removed here: d7cd2f9e511c2cad94725c47ad2b68fea5e3624c for this PR: #1799